src/main/java/report_screens/ReportScreen.java

```diff
@@ -25,15 +25,18 @@ public class ReportScreen extends JPanel implements ActionListener {
 
     JButton report, cancel;
 
+    JFrame owner;
+
     /**
-     *
+     * @param owner: main JFrame
      * @param controller
      * @param current_review
      * @param current_user
      *
      * INITIALIZE REPORT VIEW: A POP OUT WINDOW
      */
-    public ReportScreen(ReportController controller, Review current_review, User current_user) {
+    public ReportScreen(JFrame owner, ReportController controller, Review current_review, User current_user) {
+        super(owner, true);
```

Comment:
Camel case refactoring to be done,
consider renaming owner to be previousFrame or parentFrame since we have an OwnerUser calss

Reply:
just changed it